bundles/org.openhab.binding.paradoxalarm/README.md

```diff
@@ -98,14 +98,14 @@ Currently binding supports the following panels: EVO192, EVO48(not tested), EVO9
     String partition1AdditionalStates “Floor1 MUC additional states: [%s]” (Partitions) { channel = “paradoxalarm:partition:ip150:partition1:additionalStates” }
 
 //ZONES
-    Contact CorridorFl1_PIR_state “Corridor Fl1 motion: [%s]” (PIRSensors) { channel = “paradoxalarm:zone:ip150:MotionSensor1:opened” }
-    Contact CorridorFl1_MUC_state “Corridor Fl1 window: [%s]” (Floor1MUC) { channel = “paradoxalarm:zone:ip150:MagneticSensorWindow1:opened” }
+    Contact CorridorFl1_PIR_state “Corridor Fl1 motion: [%s]” (PIRSensors) { channel = “paradoxalarm:zone:ip150:MotionSensor1:isOpened” }
+    Contact CorridorFl1_MUC_state “Corridor Fl1 window: [%s]” (Floor1MUC) { channel = “paradoxalarm:zone:ip150:MagneticSensorWindow1:isOpened” }
 ```
 
 ## Example sitemap configuration
 
 ```java
-   Text label="Security" icon="lock"{
+   Sitemap Paradox label="Security" icon="lock"{

```

Comment:
The example generally provides a portion of a sitemap file, not a full sitemap.
So the original example looks good.

Reply:
I have seen both (see e.g. `avmfritz` or `mail`) and I assume both are ok.